app/controllers/api/v1/students_controller.rb

```diff
@@ -11,6 +11,22 @@ def show
     	# @skills = @student.skills
   end
 
+  def create
+    @student = Student.create({
+      :first_name => params[:first_name],
+      :last_name => params[:last_name],
+      :email => params[:email],
+      :phone_number => params[:phone_number],
+      :github => params[:github],
+      :blog => params[:blog],
+      :twitter => params[:twitter],
+      :linkedin => params[:linkedin]
+    })
+    # redirect_to api_v1_student_path(@student.id)
```

Comment:
absolutely. I'll get on it as soon as I can.

Can't we just do:
if !@student.save
render json: {errors: @spot.errors.full_messages }, status: 422
end

??

Also, I think at this point there is no sad path because there are no
validations. I'll try to add the validations as well as the sad path as
soon as I can.

How's the rest of it looking?

I won't be able to be there Wednesday night. Sorry. Would've liked to, but
it crossed paths with a higher priority.

Man Troy it's late! You gotta go get some sleep. You're always up at the
crack of dawn. How do you function on so little sleep?

Anyways, I'm definitely off to bed.

I'm looking forward for our next meeting!! Please keep me posted.

G night,

Shlomo

On Mon, May 11, 2015 at 11:12 PM, Troy <notifications@github.com> wrote:

> In app/controllers/api/v1/students_controller.rb
> <https://github.com/acltc/resume_data_api/pull/35#discussion_r30103609>:
>
> > @@ -11,6 +11,22 @@ def show
> >      	# @skills = @student.skills
> >    end
> >
> > +  def create
> > +    @student = Student.create({
> > +      :first_name => params[:first_name],
> > +      :last_name => params[:last_name],
> > +      :email => params[:email],
> > +      :phone_number => params[:phone_number],
> > +      :github => params[:github],
> > +      :blog => params[:blog],
> > +      :twitter => params[:twitter],
> > +      :linkedin => params[:linkedin]
> > +    })
> > +    # redirect_to api_v1_student_path(@student.id)
>
> I think we need to add something for the sad path such as this..
>
> if @student <https://github.com/student>.save
> else
> render json: { errors: @spot.errors.full_messages }, status: 422
> end
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/acltc/resume_data_api/pull/35/files#r30103609>.
>